routes.js

```diff
@@ -0,0 +1,327 @@
+import express from 'express';
+import path from 'path';
+import Joi from 'joi';
+import { requireJsonContent } from './middleware/index';
+import netflixData from './data/netflix-titles.json';
+
+const router = express.Router();
+const ITEMS_PER_PAGE = 20;
+let remainingPages = null;
+let totalPages = null;
+let movies = netflixData;
```

Comment:
If you're going to use `let` variables like this, they should be defined in the route - defining them in the global scope like this will lead to strange bugs or confusion if you're not careful